deploy/grafana/grafana-with-grafana-operator/grafana-dashboard-operator-cr-template.yaml

```diff
@@ -13,5 +13,7 @@ spec:
   # TODO remove this plugin definition after resolve https://github.com/integr8ly/grafana-operator/issues/155
   plugins:
     - name: "vertamedia-clickhouse-datasource"
-      # update to 2.0.4 when resolve https://github.com/grafana/grafana-plugin-repository/pull/690
+      # update to 2.1.0 when next grafana-operator release
       version: "1.9.5"
+    - name: "grafana-piechart-panel"
```

Comment:
@alex-zaitsev after these changes will merge to `master` we need to create separate separate PR to merge to `0.12.0` branch cause ACM look to 0.12.0 `grafana-dashboard-operator-cr-template.yaml`